docker/terra_image/Dockerfile

```diff
@@ -5,6 +5,8 @@ USER root
 ENV PIP_USER=false
 
 RUN pip3 install -U git+https://github.com/broadinstitute/ml4h.git
+RUN pip3 install -U tensorflow==2.4.3
```

Comment:
Can you please add a comment about when these overrides should be removed?

e.g., remove them after https://github.com/DataBiosphere/terra-docker/issues/244 ?

Thanks!

Reply:
oops, had meant to do that.  Done.

Reply:
thx!